examples/io/plot_objects_from_arrays.py

```diff
@@ -148,7 +148,7 @@
 for ai, asig in enumerate(seg.analogsignals):
     # Since the data does not contain channel names, channel indices are used.
     ch_names.append('Neo %02d' % (ai + 1,))
-    data.append(asig.rescale('V').magnitude[:, 0])
+    data.append(asig.rescale('V').magnitude)
```

Comment:
I can confirm that with latest `neo` this breaks things:
```
Traceback (most recent call last):
  File "examples/io/plot_objects_from_arrays.py", line 159, in <module>
    raw = mne.io.RawArray(data, info)
  File "<string>", line 2, in __init__
  File "/home/larsoner/custombuilds/mne-python/mne/utils.py", line 707, in verbose
    return function(*args, **kwargs)
  File "/home/larsoner/custombuilds/mne-python/mne/io/array/array.py", line 59, in __init__
    raise ValueError('Data must be a 2D array of shape (n_channels, '
ValueError: Data must be a 2D array of shape (n_channels, n_samples
```
Please revert and we should deal with this in a separate PR.